target/classes/application.properties

```diff
@@ -22,22 +28,18 @@ spring.jackson.serialization.fail-on-empty-beans=false
 # so you will have the same EntityManager until the web request is finished.
 spring.jpa.open-in-view=true
 #
-# ------------------------------------------------------------------------
-# ---------------------TESTING/LOCAL------------------------
-#spring.jpa.hibernate.ddl-auto=create
-#spring.datasource.initialization-mode=always
-#command.line.runner.enabled=true
-# ---------------------PRODUCTION/DEPLOY------------------------
-spring.jpa.hibernate.ddl-auto=update
-spring.datasource.initialization-mode=never
-command.line.runner.enabled=false
-# ------------------------------------------------------------------------
-#
 # Used to set the date format for JSON Output
 spring.jackson.date-format=yyyy-MM-dd HH:mm:ss
 spring.jackson.time-zone=America/Los_Angeles
 #
 # Turns off Spring Boot automatic exception handling
-server.error.whitelabel.enabled=false
+#server.error.whitelabel.enabled=false
 spring.mvc.throw-exception-if-no-handler-found=true
-spring.web.resources.add-mappings=false
\ No newline at end of file
+spring.web.resources.add-mappings=false
+spring.security.oauth2.resourceserver.jwt.issuer-uri=${OKTAOAUTH2ISSUER}
+okta.oauth2.client-id=${OKTAOAUTH2CLIENTID}
+okta.oauth2.issuer=${OKTAOAUTH2ISSUER}
+okta.oauth2.groups-claim=groups
+okta.client.token=00ADYFjWdYPkIMmVZwawR5jAhQIbe-mqz3uSTDc81m
+okta.client.org-url=https://dev-41327786.okta.com
```

Comment:
We probably don't want the above lines up there. I always forget about `target/`